libs/langchain-community/src/document_loaders/web/cheerio.ts

```diff
@@ -5,38 +5,27 @@ import type {
   SelectorType,
 } from "cheerio";
 import { Document } from "@langchain/core/documents";
-import {
-  AsyncCaller,
-  AsyncCallerParams,
-} from "@langchain/core/utils/async_caller";
+import { AsyncCaller } from "@langchain/core/utils/async_caller";
 import { BaseDocumentLoader } from "@langchain/core/document_loaders/base";
-import type { DocumentLoader } from "@langchain/core/document_loaders/base";
+import { WebBaseLoaderParams, WebBaseLoader } from "./html.js";
 
 /**
- * Represents the parameters for configuring the CheerioWebBaseLoader. It
- * extends the AsyncCallerParams interface and adds additional parameters
- * specific to web-based loaders.
+ * @deprecated Either import the CheerioWebBaseLoaderParams from @langchain/community/document_loaders/web/cheerio
+ * or use the WebBaseLoaderParams from @langchain/community/document_loaders/web/html.
  */
-export interface WebBaseLoaderParams extends AsyncCallerParams {
-  /**
-   * The timeout in milliseconds for the fetch request. Defaults to 10s.
-   */
-  timeout?: number;
+export { WebBaseLoaderParams };
```

Comment:
This is still technically breaking since we're removing `selector?: SelectorType;` from it but I think this is fine